### ISSUE TITLE
Next release (v2.2.0)

### ISSUE THREAD

**Issue by laupolo6** [knadh/listmonk#851]
Hi,
When will the next major/minor version be released? any guess?
I think it would be great if listmonk have a more predictable timeframe or roadmap for future releases.
thanks

**Comment by knadh** [comment]
Please see this for some additional context on release cycles and plans: https://github.com/knadh/listmonk/issues/675

That said, the next release is already at least a month behind the schedule I had in mind. Since I happen to be the only developer working on the project for now, this unfortunately entirely depends on the limited personal time I can dedicate.

v2.1.0 is a good-enough, feature complete, stable version that's in large production environments, so really, I haven't been scrambling to make a new release either. The current big WIP is transactional e-mailing and once that is done, the next version should be release. I'm hoping this will happen in the next ~2 weeks.

**Comment by sbusso** [comment]
@knadh first thank for the amazing work on listmonk. I have managed my own setup for more than 10 years, but now it is hard to maintain and listmonk is the closest to what I was doing. Is there any public roadmap or list of features you are looking at improving / implementing?

**Comment by knadh** [comment]
Thanks @sbusso. The GitHub issues are properly tagged. The ones I've picked to work on are assigned to me.

All the other open issues tagged `enhancement` are good to be discussed and worked on.

**Comment by knadh** [comment]
[Transactional mailing](https://github.com/knadh/listmonk/commit/463e92d1e1353a79a8fd8d32c75fcef5e38dee5a) is finally merged. Had to wait for a week to test the changes in production at scale. Now, a handful of pending tweaks and minor bug fixes and the release is good to go. The coming weekend :crossed_fingers:.

**Comment by NicoHood** [comment]
Can we see a fix for https://github.com/knadh/listmonk/issues/712 for the next release?

**Comment by knadh** [comment]
Tagging i18n language contributors. There are a few new strings in the English file that aren't in the other files. Would be great if you'd be able to find time to compare and submit any changes before the next release. Thank you!

- čeština (@srchlm)
- Deutsch (@jonathanmmm @an0nfunc @spezifisch @dunklesToast @ChrisTG742 @m42e @NicoHood)
- Español (@jorge-vitrubio @dreglad @seba81)
- Français (@nathanaelhoun @jonathandhn @MickGe @FelixDz @jonathandhn)
- Hungarian (@bohemtucsok)
- Italiano (@etcware @jorge-vitrubio @marcofucito @TomBoss)
- 英語 (@t3hmrman)
- മലയാളം (@joicemjoseph)
- Nederlands (@TychoWerner @Jjagg @sjoerdvanderhoorn)
- Polski (@marcinkunert)
- Português Brasileiro (@enniosousa @rslonik)
- Portuguese (@diogotcorreia)
- Română (@gabrielpioaru)
- Русский (@inpos)
- Suomi (@pucilpet)
- Turkish (@senol)
- Vietnamese (@mannm123)
- 名字 (@p1slave)

**Comment by t3hmrman** [comment]
> 英語 (@t3hmrman)

That says "English" (in japanese), I think what you want there is "Japanese" I'm assuming, which would be:

> 日本語 (@t3hmrman)

Thanks again for getting that into the release and as always thanks for the awesome software.

**Comment by knadh** [comment]
Whoops. That's how it is in the [jp file](https://github.com/knadh/listmonk/blob/0cd41ed9c4b4df57a482953dd79afd3adb49c0a8/i18n/jp.json#L3)! Will fix this.

**Comment by t3hmrman** [comment]
Oh apologies that's actually my fault -- will include it in this PR:

https://github.com/knadh/listmonk/pull/869

**Comment by t3hmrman** [comment]
Also one more thing -- for the project would you consider disabling merge commits? It's a bit of personal preference but I think it's nice to have all PRs get squashed.

Obviously a big change (others might disagree) but I think it's a nice default. If it helps, it does match listmonk's stated contribution guidelines:

> PRs will be squashed in the end: A PR may change considerably with multiple commits before it is approved. Once a PR is approved, if there are multiple commits, they will be squashed into a single commit during merging.

**Comment by knadh** [comment]
Ah yes, sorry. This was a mistake. I do squash commits when merging, but overlooked it in #869.

**Comment by t3hmrman** [comment]
Oh yeah no problem -- I forgot to squash it myself as well! Disabling merge commits would probably be a great way to fix the problem for good (unless merge commits are useful to the project for some other reason)!

**Comment by rslonik** [comment]
Done, created a PR:
https://github.com/knadh/listmonk/pull/875

**Comment by MickGe** [comment]
Done, created a PR #877.

**Comment by pucilpet** [comment]
Done, #878

**Comment by knadh** [comment]
[v2.2.0](https://github.com/knadh/listmonk/releases/tag/v2.2.0) is released. Thanks everyone.
